slices/gawk.yaml

```diff
@@ -0,0 +1,27 @@
+package: gawk
+
+slices:
```

Comment:
nit: It would also be nice to have the slice definitions in a lexicographic order based on slice names. So `bins`, `headers`, then `libs`.